modules/microshift-disabling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+// * microshift_configuring/microshift-disable-lvms-csi-provider-csi-snapshot.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-disabling-lvms-csi-snapshot_{context}"]
+= Disabling deployments that run CSI snapshot implementations
+
+Before you disable the CSI snapshot implementation pods, you must manually stop any user workloads that have attached volumes. Use the following procedure to disable installation of the CSI implementation pods.
+
+[NOTE]
+====
+{microshift-short} will not delete CSI snapshot implementation pods. You must configure {microshift-short} to disable installation of the CSI snapshot implementation pods during the startup process.
+====
+
+.Procedure
+
+. Disable installation of the CSI snapshot controller by entering the `optionalCsiComponents` value under the `storage` section of the {microshift-short} configuration file in `/etc/microshift/config.yaml`:
++
+[source,yaml]
+----
+# ...
+  storage
+   optionalCsiComponents: "snapshot-controller" <1>
```

Comment:
This is not constructed properly and will cause a failure when the microshift service is restarted

```suggestion
   optionalCsiComponents: 
        - "snapshot-controller" <1>
```

Reply:
updated for both "microshift-disabling-lvms-csi-snapshot.adoc" and "microshift-disabling-lvms-csi-driver.adoc"